nuntium/templates/nuntium/message/message_detail.html

```diff
@@ -17,7 +17,9 @@ <h1 class="panel-title">{{ message.subject }}</h1>
     </div>
   </div>
   <div class="panel-footer row">
-    <span class="author_name col-md-4"><small>{% blocktrans with message.author_name as name %}Asked by {{ name }}.{% endblocktrans %}</small></span>
+    {% url 'all-messages-from-the-same-author-as' slug=message.writeitinstance.slug message_slug=message.slug as other_messages_by %}
+    {% blocktrans with message.author_name as name %}Asked by 
+    <a href="{{other_messages_by}}">{{ name }}</a>.{% endblocktrans %}</small></span>
```

Comment:
I think you'll need to put `other_messages_by` into the `with` in the `blocktrans` too.